glean-core/ffi/examples/Makefile

```diff
@@ -1,4 +1,4 @@
-objects = glean_app.c ../../../target/release/libglean_ffi.dylib
+objects = glean_app.c ../../../target/release/libglean_ffi.so
```

Comment:
Ah yeah, we probably need to properly detect what system we're on.

Reply:
I'm just handling this with the use of `$(wildcard )` now which isn't really detecting the system, but should be good enough for now...

Reply:
hm, probably still breaks on Windows, but good enough for now I'd say

Reply:
Yeah, but it never worked on Windows ;)  Maybe we leave this to @Dexterp37 to decide whether to handle now (or later)...